Subject: Don't touch rss-glib if RSS is disabled

rss-glib is licensed under the LGPLv3 or later.  src/hig.c, src/hig.h,
src/torrent-cell-renderer.c, and src/torrent-cell-renderer.h are licensed under
the GPLv2 only.  Therefore, transmission-remote-gtk must be built without RSS to
avoid a GPL violation.

Disabling RSS with --without-libmrss should be enough to ensure this happens.
In the interest of defence in depth, however, conditionalize references to
rss-glib from within the build system.  As an added bonus, this speeds up build
time by not building code which will never get linked.

Author: Benjamin Barenblat <bbaren@mit.edu>
Bug: https://github.com/transmission-remote-gtk/transmission-remote-gtk/issues/21
--- a/meson.build
+++ b/meson.build
@@ -85,7 +84,6 @@
 conf_data.set('G_DISABLE_CAST_CHECKS', not debug)
 
 # compile time features
-conf_data.set10('HAVE_RSS', rss_dep.found())
 conf_data.set10('HAVE_GEOIP', geoip_dep.found())
 conf_data.set10('HAVE_LIBPROXY', libproxy_dep.found())
 conf_data.set10('ENABLE_NL_LANGINFO', nl_langinfo)
@@ -135,7 +133,4 @@
 
 subdir('po')
 subdir('data')
-if rss_dep.found()
-  subdir('extern/rss-glib')
-endif
 subdir('src')
